store/snackbar.ts

```diff
@@ -15,6 +15,6 @@ interface MutationPayload {
 export const mutations: MutationTree<SnackbarState> = {
   display(state: SnackbarState, payload: MutationPayload) {
     state.message = payload.message;
-    state.color = !payload.color ? <string>payload.color : '';
+    state.color = payload.color ? <string>payload.color : '';
```

Comment:
```suggestion
    state.color = Boolean(payload.color) ? <string>payload.color : '';
```
Untested but should work